linode/manager
# Update `launchdarkly-react-client-sdk` to `latest`

## Description 📝

- Remove Launchdarkly Patch because it is no longer needed
- Disable postinstall command because we don't need to do any patching
- Updates  `launchdarkly-react-client-sdk` to `latest`

## How to test 🧪

- Have `REACT_APP_LAUNCH_DARKLY_ID` defined in your `.env` (ask me for this value)
- Re-run `yarn dev` to pick up new dependencies and env value
- Verify Launch Darkly is working by 
  - Logging into Launch Darkly
  - Having Cloud Manger open
  - Toggling a flag like `Database Beta` **in the Test environment**
  - Making sure that Cloud Manager updated to reflect the new value
